desktop/libs/indexer/src/indexer/api3_tests.py

```diff
@@ -0,0 +1,26 @@
+import tempfile
+import uuid
+from django.http import HttpRequest, QueryDict
+from indexer.api3 import guess_field_types
+from nose.tools import assert_equal, assert_true
+
+
+def test_local_file_header_reformatting_with_header():
+    with open('apps/beeswax/data/tables/testbook1.csv', encoding='utf-8-sig') as upload_file:
```

Comment:
Could we do something similar to, e.g.

with opent(get_desktop_root('apps', 'beeswax', 'data', 'tables', 'testbook1.csv')) ...

`from desktop.lib.paths import get_desktop_root`